app/src/main/java/com/vinh/dictionary_1/screen/home/HomeViewModel.java

```diff
@@ -128,4 +130,28 @@ public void setTextWatcher(TextWatcher textWatcher) {
         mTextWatcher = textWatcher;
         notifyPropertyChanged(BR.textWatcher);
     }
+
+    public void onSearchWordEditTextTouch() {
+        if (mWordListFragment == null) {
+            mWordListFragment = WordListFragment.newInstance();
+            setTextWatcher(mWordListFragment);
+        }
+        setFragment(mWordListFragment);
+    }
+
+    @Override
+    public boolean onTouch(View view, MotionEvent motionEvent) {
```

Comment:
WARNING: [ClickableViewAccessibility] `HomeViewModel#onTouch` should call `View#performClick` when a click is detected
 If a `View` that overrides `onTouchEvent` or uses an `OnTouchListener` does not also implement `performClick` and call it when clicks are detected, the `View` may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in `View#performClick` as some accessibility services invoke `performClick` when a click action should occur.